providers/defaults.go

```diff
@@ -552,4 +552,18 @@ var DefaultProviders Providers = map[string]*Provider{
 			},
 		},
 	},
+	"ipinfo": {
+		Provider: &plugin.Provider{
+			Name:            "ipinfo",
+			ID:              "go.mondoo.com/cnquery/v12/providers/ipinfo",
```

Comment:
We should change the provider to `go.mondoo.com/cnquery/providers/ipinfo` and I think we should adjust the scaffolding to generate the new scheme out of the box but we should do that a separate PR